v8/db.go

```diff
@@ -883,6 +883,44 @@ func (db *DB) SetOptions(keys, values []string) error {
 	return nil
 }
 
+// SetOptionsCF dynamically changes options through the SetOptions API for specific Column Family.
+func (db *DB) SetOptionsCF(cf *ColumnFamilyHandle, keys, values []string) (err error) {
+	numKeys := len(keys)
+	if numKeys == 0 {
+		return nil
+	}
+
+	cKeys := make([]*C.char, numKeys)
+	cValues := make([]*C.char, numKeys)
+	for i := range keys {
+		cKeys[i] = C.CString(keys[i])
+		cValues[i] = C.CString(values[i])
+	}
+
+	var cErr *C.char
+
+	C.rocksdb_set_options_cf(
+		db.c,
+		cf.c,
+		C.int(numKeys),
+		&cKeys[0],
+		&cValues[0],
+		&cErr,
+	)
+	if cErr != nil {
+		err = errors.New(C.GoString(cErr))
+		C.rocksdb_free(unsafe.Pointer(cErr))
+	}
+
+	// free before return
```

Comment:
I noticed the `SetOptions` function does not free the cstrings. Is that a bug or do we not need to free these?

Reply:
I think we probably do need to technically. Although I don't think it is super high impact since we would not be continuously setting options.

Might be a better idea to make a separate PR to fix that, though